src/test/java/org/mule/module/apikit/odata/ODataUriParserTestCase.java

```diff
@@ -4790,4 +4790,12 @@ public void getOdataUrlTest() {
 
     assertEquals(odataUrl, ODataUriHelper.getOdataUrl(completeUrl));
   }
+
+  @Test
+  public void validLongQuerySelect() throws ODataException {
```

Comment:
Add another test with an invalid value

Reply:
there are several already included at the test 
https://github.com/mulesoft/apikit-odata-extension/blob/405f79b361b981ec6984a24dcc7191f8ead01f7e/src/test/java/org/mule/module/apikit/odata/ODataUriParserTestCase.java#L599-L618